owenhallgren/stretch-project
# Feature/dashboard view

## What does this PR do?
This adds a currentReviews component, the component has a table that holds the users reviews that they have accepted. The user can now also add a review from the open reviews page. 

## How should it be tested in the UI?
Run the app and make sure the currentReviews component responds well in a mobile view
